cpp/cmake/thirdparty/get_faiss.cmake

```diff
@@ -66,7 +68,22 @@ function(find_and_configure_faiss)
     rapids_export_find_package_root(BUILD faiss [=[${CMAKE_CURRENT_LIST_DIR}]=] raft-nn-lib-exports)
 endfunction()
 
+if(NOT RAFT_FAISS_GIT_TAG)
+  # TODO: Remove this once faiss supports FAISS_USE_CUDA_TOOLKIT_STATIC
```

Comment:
Is the plan to wait for this before we merge this PR? Or should we create a RAFT issue just to make sure it stays on our radar?

Reply:
@cjnolet it's up to you -- I based this branch from the commit sha here. If pointing at my fork is fine with you, then great. Otherwise, it might be worth forking faiss into rapidsai and cherry-picking my commit.